cache_run_c_code = 'Test does not build: Traceback (most recent call last):\n  File "/home/trin/programming/switchless-sim/ns-allinone-3.16/ns-3.16/.waf-1.6.11-30618c54883417962c38f5d395f83584/waflib/Tools/c_config.py", line 445, in run_c_code\n    bld.compile()\n  File "/home/trin/programming/switchless-sim/ns-allinone-3.16/ns-3.16/.waf-1.6.11-30618c54883417962c38f5d395f83584/waflib/Build.py", line 190, in compile\n    raise Errors.BuildError(self.producer.error)\nBuildError: Build failed\n -> task in \'testprog\' failed (exit status 1): \n\t{task 173402540: cxxprogram test.cpp.1.o -> testprog}\n[\'/usr/bin/g++\', \'-Wl,-Bsymbolic-functions\', \'-Wl,-z,relro\', \'test.cpp.1.o\', \'-o\', \'/home/trin/programming/switchless-sim/ns-allinone-3.16/ns-3.16/build/.conf_check_9a1d0551212c9d703363e62dfbc51c79/testbuild/testprog\', \'-Wl,-Bstatic\', \'-Wl,-Bdynamic\', \'-L/usr/lib\', \'-lpython2.7\']\n'
